.github/workflows/debian9.yml

```diff
@@ -37,23 +39,30 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-# I386: actions/checkout@V2,  actions/cache@V2 actions/cache@V2 
-#       give error about OCI container [https://github.com/actions/checkout/issues/334]
-#                                      [https://github.com/actions/runner/issues/1011]
         env:
-          - CPU: i386
+          - CC: gcc
+            CXX: g++
+            CPU: i386
+            IMAGE: "i386/debian:9"
+          - CC: clang
+            CXX: clang++
+            CPU: i386
             IMAGE: "i386/debian:9"
     env: ${{ matrix.env }}
+    name: ${{ matrix.env.IMAGE }} tests ${{ matrix.env.CC }}
     steps:
       - name: Install prerequisites
         run: |
           apt update
           apt -y install git sudo
       
+# I386: actions/checkout@V2,  actions/cache@V2 actions/cache@V2
+#       give error about OCI container [https://github.com/actions/checkout/issues/334]
+#                                      [https://github.com/actions/runner/issues/1011]
       - name: Check out repository
         uses: actions/checkout@v1
         with:
-          fetch-depth: 1
+          fetch-depth: 10
```

Comment:
There is some issue with actions/checkout@1 causing problems with fetch-depth: 1,2,3 ...
I was not able to find any quick explanation or support but fetch-depth: 10 works